Makefile

```diff
@@ -1,11 +1,16 @@
 run:
-	@python hr-data-analytics-package-folder/main.py
+	@python hr_data_analytics_package_folder/main.py
```

Comment:
Due to name conflict on Google Artifact Registry, I had to change our folder name from hyphens (-) to underscores ( _ ).